collect_app/src/main/java/org/odk/collect/android/utilities/CustomTabHelper.java

```diff
@@ -24,7 +24,7 @@
 public class CustomTabHelper {
     public static final String OPEN_URL = "url";
     private static final String CUSTOM_TAB_PACKAGE_NAME = "com.android.chrome";
-    private CustomTabsClient customTabsClient;
+    public CustomTabsClient customTabsClient;
```

Comment:
Is this an accidental change? I can't see a reason for it.

Reply:
Oh Sorry, I was trying something in `onDetachedFromWindow` test.